opm/simulators/wells/WellFilterCake.cpp

```diff
@@ -0,0 +1,129 @@
+/*
+  Copyright 2017 SINTEF Digital, Mathematics and Cybernetics.
+  Copyright 2017 Statoil ASA.
```

Comment:
Maybe the copyright claim can stay  with `  Copyright 2023 Equinor.`

Reply:
of course, didn't know what to put since this code is so new. changed.